### PR TITLE
Tooltip

### PR DESCRIPTION
**Pull request by jmharris229**
This pull request:
1.Adds dependencies for bootstrap and Jquery, within the lib folder
2.Tests basic functionality of tooltips/popovers
3. updated the readme for feature delegation
4. reintegrated the highlight feature that was commented out in the last pull request. 
5. Switched the popup from the previous request to a side panel. 

After playing with popovers and tooltips, i think we will probably go with a side bar due to it providing more space, cleaner, and less clustered look to the page. Plus, it will be easier to inject dynamic data.

### CODE REVIEW

**Comment by franticallycodes** [comment]
:+1:
@AdamSheaffer, @BlaiseGratton, @jcockhren Will you evaluate this PR please? Thanks!

File: README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ E2 Group Project for implementing an app for annotating text on a web page.
 Purpose - Allow a user to add notes or Anotation to a selected web page
 
 **Features**
-- The ability to create a new user account, and or login as a user
-- The ability to add comments to pre existing anotations
-- The ability to select a highlight color from a set pallet of choices
-- The ability to share Anotations via Facebook
-- The ability to pull info on the highlighted text from the wiki API
+- The ability to create a new user account, and or login as a user(martin)
```
Comment (BlaiseGratton) [inline README.md]: Martin? (capitalized)

File: js/main.js [hunk 2]
```diff
@@ -1,42 +1,21 @@
...
-    if (window.getSelection) {
-        text = window.getSelection().toString();
+    if (document.getSelection) {
+        var text = document.getSelection()
+        var textsel = text.toString();
+        console.log(textsel);
+        var selectionNode = text.anchorNode.parentElement.innerHTML;
+        var markedText = selectionNode.replace(textsel, "<mark class='"+text+"' data-toggle='popover' title='hello' data-placement='left' data-content='...'>"+textsel+"</mark>");
+        $('.paragraph1').html(markedText);
+        $()
```
Comment (BlaiseGratton) [inline js/main.js]: Can this empty selector be removed?

**Comment by BlaiseGratton** [comment]
Haha! Thumbs up from me now

**Pull request merged**